set.mm

```diff
@@ -62489,20 +62489,45 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
 
   ${
     $d x A $.  $d x F $.
-    $( A function's value at a proper class is the empty set.  (Contributed by
-       NM, 20-May-1998.) $)
+    $( A function's value at a proper class is the empty set.  See ~ fvprcALT
+       for a proof that uses ~ ax-sep and ~ ax-pr instead of ~ ax-pow .
+       (Contributed by NM, 20-May-1998.) $)
     fvprc $p |- ( -. A e. _V -> ( F ` A ) = (/) ) $=
       ( vx cvv wcel wn cv wbr weu cfv c0 wceq brprcneu tz6.12-2 syl ) ADEFACGBH
       CIFABJKLCABMCABNO $.
   $}
```

Comment:
So we can add "$( $j usage 'fvprc' avoids 'ax-sep' 'ax-pr' ; $)" here?

Reply:
Done.